### PR TITLE
Design implementation of NFT info page

### PR DESCRIPTION
**Pull request by krisutofu**
I didn't implement a small screen version yet. For small screen versions, I just would put the NFT-specific info column (on the right) between the image and the drop information on the left side.

No functionality (fetching data) is implemented except for switching the group size of the NFT card container at the bottom. Copying to clipboard works as well with older browser support.

Functionality of the NFT info page is another whole user story, maybe even two.

I implemented the designs for the different scenarios which can be seen by setting individual variables to null, false or non-null, true.

The price and "place order" button is not always shown. It's hidden if there is no price (if NFT is not for sale at the moment).

nft_info_foreign_user
nft_info_page_admin

### CODE REVIEW

File: frontend/src/pages/NFTInfoPage.js [hunk 1]
```diff
@@ -0,0 +1,570 @@
...
+		Subtitle or description, can be in two rows.
+		Or even longer description can be here or somewhere else.
+		Subtitle or description, can be in two rows.
+		Or even longer description can be here or somewhere else.<br/><br/>
+
+		Subtitle or description, can be in two rows.<br/><br/>
+
+		Or even longer description can be here or somewhere else.
+		Subtitle or description, can be in two rows.
+		Or even longer description can be here.
```
Comment (dominic-heil) [inline frontend/src/pages/NFTInfoPage.js:490]: For now we should add some text here where we just fill in some infromation from the drop as we do not have a way to set a custom description atm

**Review by dominic-heil — APPROVED** [review]
Nothing major. LGTM

File: frontend/src/components/CodeTypography.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import React from "react";
+import { textColor } from "../assets/jss/colorPalette";
+
+/**
+ * Typography used for (source) code and monospace texts. Builds up on MUI Typography.
+ * @param props any props that are passed to the MUI Typography
+ * @returns {JSX.Element}
+ */
+export default function CodeTypography(props){
+	return <Typography {...props} style={{ fontFamily: "Roboto Mono, Noto Sans Mono, monospace", color: textColor,  ...props.style }}>
```
[the remark this answers is not in the document]
Comment (krisutofu) [reply frontend/src/components/CodeTypography.js:15]: Thank you! That was the reason :-D . I didn't understand it.

**Comment by krisutofu** [comment]
I took a long time to think about how I want to do the changes. I set the visibility of the content editing buttons to "hidden".

**Comment by krisutofu** [comment]
Could someone verify whether the Wallet works? Changing Password works for me.

When I connect the wallet, I get an AppwriteException that I cannot execute a task for role "" because my role is "member". Seems like it expects wrong permissions on my side.

File: frontend/src/components/NftCardStructuredList.js [hunk 3]
```diff
@@ -10,15 +11,69 @@ import { activeTextColor, textColor } from "../assets/jss/colorPalette";
...
+		<div style={{ position: "absolute", right: 0 }} >
+			<Grid container direction="row">
+				<img src={GroupLargeIconSvg} alt="Large Icons" onClick={() => setSelectedGroupSize("large")} style={{ ...groupSizeSymbolStyle(GroupSizes.large), cursor: "pointer" }}/>
+				<Grid item style={{ width: "9px", }}/>
+				<img src={GroupSmallIconSvg} alt="Small Icons" onClick={() => setSelectedGroupSize("small")} style={{ ...groupSizeSymbolStyle(GroupSizes.small), cursor: "pointer" }}/>
+			</Grid>
+		</div>
+		{children}
+	</div>);
+}
```
[the remark this answers is not in the document]
Comment (dominic-heil) [reply frontend/src/components/NftCardStructuredList.js]: Yeah, the components are only used twice now afaik. Let's refactor this in the next sprint or so.

**Review by Floinko — APPROVED** [review]
Since functionality was not part of this ticket, it looks fine to me.

File: frontend/src/pages/NFTInfoPage.js (hunk 1 above)
Comment (krisutofu) [reply frontend/src/pages/NFTInfoPage.js:498]: I put your advice in a TODO comment there.

**Pull request merged**